packages/common/test/config/test-collections.yml

```diff
@@ -9,7 +9,7 @@ providers:
         activity: SIPSPROVDownloadActivity
         conn_config:
           conn_type: ftp
-          host: localhost
+          host: 127.0.0.1
```

Comment:
In Travis CI `localhost` can resolve to an IPv6 address, but the docker containers are only listening on the IPv4 address.